constraints.pro

```diff
@@ -233,11 +233,6 @@ gen_enforced_dependency(WorkspaceCwd, DependencyIdent, null, DependencyType) :-
   workspace_has_dependency(WorkspaceCwd, DependencyIdent, DependencyRange, DependencyType),
   DependencyType \= 'dependencies'.
 
-% eth-query has an unlisted dependency on babel-runtime, so that package needs
```

Comment:
As this comment indicates, this dependency was not unused. We've tried to remove this before, but had to bring it back (see https://github.com/MetaMask/controllers/pull/341)